packages/contracts/src/dollar/core/CreditNFTManager.sol

```diff
@@ -389,9 +398,12 @@ contract CreditNFTManager is ERC165, IERC1155Receiver {
                 manager.getExcessDollarsDistributor(address(this))
             );
             // transfer excess dollars to the distributor and tell it to distribute
-            dollarToken.transfer(
-                manager.getExcessDollarsDistributor(address(this)),
-                excessDollars
+            require(
+                dollarToken.transfer(
+                    manager.getExcessDollarsDistributor(address(this)),
+                    excessDollars
+                ),
+                "UbiquityDollar: Transfer failed"
```

Comment:
```suggestion
                "Dollar: Transfer failed"
```

I feel that writing "Ubiquity" is redundant within the Ubiquity Dollar codebase so I err on the side of removing when I see "Ubiquity"